tests/base-notebook/test_container_options.py

```diff
@@ -78,3 +78,24 @@ def test_unsigned_ssl(
     assert "ERROR" not in logs
     warnings = TrackedContainer.get_warnings(logs)
     assert not warnings
+
+
+def test_custom_internal_port(
```

Comment:
Let's add `pytest.mark.parametrize` almost the same way as it is done in test_health.

Reply:
Added with [e0fda89](https://github.com/jupyter/docker-stacks/pull/1859/commits/e0fda8994849369c61869a7e4851006b36e9e1ab)